docs/UserGuide.md

```diff
@@ -111,12 +111,12 @@ Format: `help`
 
 Adds a contact.
 
-Format: `addc n/CONTACTNAME p/PHONENUMBER e/EMAIL u/GIT_USERNAME [t/TAGS]`
+Format: `addc n/CONTACTNAME p/PHONENUMBER e/EMAIL u/GITHUB_USERNAME [t/TAGS]`
 
 Examples:
 
 * `addc n/john p/12345678 e/john@nus.edu.sg u/john123 t/Schoolmate`
-* `addc n/mary p/87654321 e/mary@nus.edu.sg u/maryCS t/Teammate t/Classmate`
+* `addc n/mary p/87654321 e/mary@nus.edu.sg u/maryCS t/Tea![img.png](img.png)Hmmate t/Classmate`
```

Comment:
I think you accidentally edited this?

Reply:
Ahh I think i put in the wrong one in the merging, will change it